app/services/amr/single_read_converter.rb

```diff
@@ -0,0 +1,52 @@
+module Amr
+  class SingleReadConverter
+    def initialize(single_reading_array)
+      @single_reading_array = single_reading_array
+      @results_array = []
+    end
+
+    def perform
```

Comment:
There is an update to this in a later PR as I forgot to update the meter id - also I think there is probably better ways of doing this!

Reply:
https://github.com/BathHacked/energy-sparks/pull/774/files